app/containers/Applications/ApplicationDetailsContainer.tsx

```diff
@@ -156,6 +156,11 @@ export const ApplicationDetailsComponent: React.FunctionComponent<Props> = (
             formResult={node}
             query={props.query.query}
             showDiff={showDiff}
+            applicationRevision={
+              props.review
+                ? props.application.latestSubmittedRevision
+                : props.application.latestDraftRevision
+            }
```

Comment:
This was heavily refactored in #1585 , where I also added an `applicationRevision` fragment to `ApplicationDetailsCardItem`. Re. conflict resolution, I think that it would be easier to get #1585 merged first